utils/rpms/daos.spec

```diff
@@ -162,6 +163,12 @@ echo "%{_libdir}/daos_srv" > %{?buildroot}/%{_sysconfdir}/ld.so.conf.d/daos.conf
 %{_libdir}/libdaos_common.so
 # TODO: this should move to %{_libdir}/daos/libplacement.so
 %{_libdir}/daos_srv/libplacement.so
+# Certificate generation files
+%{daoshome}/certgen/admin.cnf
+%{daoshome}/certgen/agent.cnf
+%{daoshome}/certgen/ca.cnf
+%{daoshome}/certgen/server.cnf
+%{daoshome}/certgen/gen_certifictes.sh
```

Comment:
You can use:

```
%{daoshome}/certgen/
```

ere in place of all of those entries if you want to future-proof adding more files to this dir